server/src/test/scala/services/crunch/CrunchGraphInputsAndProbesStageFunctionsSpec.scala

```diff
@@ -1,117 +0,0 @@
-package services.crunch
-
-import drt.shared._
-import services.graphstages.Crunch._
-import uk.gov.homeoffice.drt.ports.PaxTypes.EeaMachineReadable
-import uk.gov.homeoffice.drt.ports.Terminals.T1
-import uk.gov.homeoffice.drt.ports.{PortCode, Queues}
-
-import scala.collection.immutable.List
-
-
-class CrunchGraphInputsAndProbesStageFunctionsSpec extends CrunchTestLike {
```

Comment:
are these test covered ?

Reply:
Yes the class being tested is no longer used. I forgot to remove the class, but will now